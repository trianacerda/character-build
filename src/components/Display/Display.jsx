export default function Display({ phrases }) {
  return (
    <div>
      {phrases.map((phrase) => (
        <p key={phrase}>{phrase}</p>
      ))}
    </div>
  );
}
