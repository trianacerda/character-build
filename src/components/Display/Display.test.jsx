import { render } from '@testing-library/react';
import Display from './Display';

it('should render Display element on page', () => {
  const { container } = render(<Display phrases={['hey there', 'yellowstone']} />);
  expect(container).toMatchSnapshot();
});
