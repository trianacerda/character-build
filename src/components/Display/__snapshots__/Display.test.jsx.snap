// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Display element on page 1`] = `
<div>
  <div>
    <p>
      hey there
    </p>
    <p>
      yellowstone
    </p>
  </div>
</div>
`;
